src/main/java/com/gregtechceu/gtceu/api/gui/misc/IGhostItemTarget.java

```diff
@@ -58,6 +59,10 @@ default Object convertIngredient(Object ingredient) {
                 itemStack.setTag(itemEmiStack.getNbt());
             }
         }
+
+        if (LDLib.isJeiLoaded() && ingredient instanceof ITypedIngredient<?> itemJeiStack) {
+            return itemJeiStack.getItemStack().orElse(null);
```

Comment:
Do NOT do null item stacks.
Use `ItemStack.EMPTY`

Reply:
okay, have replaced null with ```ItemStack:.Empty```